config/pipeline.yaml

```diff
@@ -342,9 +361,91 @@ platforms:
 
   shell:
 
+  kontron-kswitch-d10-mmt-6g-2gs:
+    mach: at91
+    class: arm-dtb
+    dtb: 'lan966x-kontron-kswitch-d10-mmt-6g-2gs.dtb'
+    boot_method: uboot
+
+  kontron-kswitch-d10-mmt-8g:
+    mach: at91
+    class: arm-dtb
+    dtb: 'lan966x-kontron-kswitch-d10-mmt-8g.dtb'
+    boot_method: uboot
+
+  imx6dl-riotboard:
+    mach: imx
+    class: arm-dtb
+    boot_method: barebox
+    filters:
+      - blocklist:
+          defconfig:
+            - 'imx_v4_v5_defconfig'
+            - 'multi_v5_defconfig'
+            - 'allmodconfig'
+
+  imx6qp-wandboard-revd1:
+    mach: imx
+    class: arm-dtb
+    boot_method: barebox
+    flags: ['fastboot']
+    filters:
+      - blocklist:
+          defconfig:
+            - 'imx_v4_v5_defconfig'
+            - 'multi_v5_defconfig'
+            - 'allmodconfig'
+
+  dove-cubox:
+    mach: mvebu
+    class: arm-dtb
+    boot_method: barebox
+    filters:
+      - blocklist:
+          defconfig:
+            - 'allmodconfig'
+            - 'allnoconfig'
+            - 'tinyconfig'
+            - '-chromebook'
 
 scheduler:
 
+  - job: baseline-armel-kontron-android_mainline
+    event:
+      channel: node
+      name: android-mainline
```

Comment:
You must actually define a `kbuild` job (e.g. `kbuild-gcc-10-armel-android-mainline`), add a corresponding scheduler entry, and use that job's name here.

Reply:
right, so what is the meaning of the `build_configs:` if they are not used? (At least my grep powers failed me)

Reply:
IIUC it *is* used but only by a specific service, @nuclearcat should have the details about that.

Reply:
I guess it is just a way to say to kernelci to monitor this tree (I guess)